<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSCEpisodePreparation</class>
 <widget class="QWidget" name="LSCEpisodePreparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSCEpisodePreparation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbSt1">
       <property name="text">
        <string>STATE1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSt2">
       <property name="text">
        <string>STATE2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSt3">
       <property name="text">
        <string>STATE3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSt4">
       <property name="text">
        <string>STATE4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSt5">
       <property name="text">
        <string>STATE5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
